src/api/app/models/workflow/step.rb

```diff
@@ -111,6 +111,10 @@ def create_subscriptions(package, workflow_filters)
     end
   end
 
+  def destroy_subscriptions(package, _workflow_filters)
```

Comment:
I don't see why we would ever need workflow filters when deleting `EventSubscription` records.

```suggestion
  def destroy_subscriptions(package)
```

Reply:
done